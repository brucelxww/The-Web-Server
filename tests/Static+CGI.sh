cd ..
python3 webserv.py config.cfg &
PID=$!
cd -
sleep 2
curl -i localhost:8070/home.js > staticcgioutput
sleep 2
curl -i localhost:8070/info1112.txt >> staticcgioutput
sleep 2
curl -i localhost:8070/cgibin/hello.py >> staticcgioutput
sleep 2
curl -i localhost:8070/ >> staticcgioutput
diff staticcgioutput Static+CGI.out
kill $PID