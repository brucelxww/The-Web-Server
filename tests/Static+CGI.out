HTTP/1.1 200 OK
Content-Type: application/javascript


function hello() {
	console.log("Hello from javascript!");
}
HTTP/1.1 200 OK
Content-Type: text/plain

lalalallalaHTTP/1.1 200 OK
Content-Type: text/html

<h1>Hello From Python!</h1>
HTTP/1.1 200 OK
Content-Type: text/html

<html>
<head>
	<title>Hello!</title>
</head>
<body>
<h1>Hello World!</h1>
</body>
</html>
